src/Task/ComposerNormalize.php

```diff
@@ -52,24 +53,43 @@ public function run(ContextInterface $context): TaskResultInterface
         }
 
         $arguments = $this->processBuilder->createArgumentsForCommand('composer');
-        $arguments->add('normalize');
+        $arguments = $this->addArgumentsFromConfig($arguments, $config);
         $arguments->add('--dry-run');
 
-        if ($config['indent_size'] !== null && $config['indent_style'] !== null) {
-            $arguments->addOptionalArgument('--indent-style=%s', $config['indent_style']);
-            $arguments->addOptionalArgument('--indent-size=%s', $config['indent_size']);
-        }
-
-        $arguments->addOptionalArgument('--no-update-lock', $config['no_update_lock']);
-        $arguments->addOptionalArgument('-q', $config['verbose']);
-
         $process = $this->processBuilder->buildProcess($arguments);
         $process->run();
 
         if (!$process->isSuccessful()) {
-            return TaskResult::createFailed($this, $context, $this->formatter->format($process));
+            $output = $this->formatter->format($process);
+            $arguments = $this->processBuilder->createArgumentsForCommand('composer');
```

Comment:
Instead of rebuilding, you could also remove the `--dry-run` option from the main process (that might be easier)